usaspending_api/awards/v2/lookups/lookups.py

```diff
@@ -0,0 +1,1122 @@
+transaction_d1_columns = {
```

Comment:
settings.py has a `LONG_TO_TERSE` mapping for column names, can that be leveraged here?

Reply:
This can be ignored, keeping the lookups here is appropriate.